src/runtime_src/doc/toc/xrt_native_apis.rst

```diff
@@ -16,39 +16,39 @@ Example g++ command
-|   Kernel      | xrt::kernel   |  xrtKernelHandle  |
-+---------------+---------------+-------------------+
-|   Run         | xrt::run      |  xrtRunHandle     |
-+---------------+---------------+-------------------+
-|   Graph       | TBD           |  xrtGraphHandle   |
-+---------------+---------------+-------------------+
 
-All the core data structures are defined inside in the header files at ``$XILINX_XRT/include/experimental/`` directory. In the user host code, it is sufficient to include ``"experimental/xrt_kernel.h"`` and ``"experimental/xrt_aie.h"`` (when using Graph APIs) to access all the APIs related to these data structure.
+The C++ Class objects used for the APIs are 
 
-.. code:: c
-      :number-lines: 5
-           
-           #include "experimental/xrt_kernel.h"
-           #include "experimental/xrt_aie.h"
++----------------------+-------------------+------------------------------------------------+
+|                      |   C++ Class       |  Header files                                  |
++======================+===================+================================================+
+|   Device             | ``xrt::device``   |  ``#include <xrt/xrt_device.h>``               |
++----------------------+-------------------+------------------------------------------------+
+|   XCLBIN             | ``xrt::xclbin``   |  ``#include <experimental/xrt_xclbin.h>``      |
++----------------------+-------------------+------------------------------------------------+
+|   Buffer             | ``xrt::bo``       |  ``#include <xrt/xrt_bo.h>``                   |
++----------------------+-------------------+------------------------------------------------+
+|   Kernel             | ``xrt::kernel``   |  ``#include <xrt/xrt_kernel.h>``               |
++----------------------+-------------------+------------------------------------------------+
+|   Run                | ``xrt::run``      |  ``#include <xrt/xrt_kernel.h>``               |
++----------------------+-------------------+------------------------------------------------+
+| User-managed Kernel  | ``xrt::ip``       |  ``#include <experimental/xrt_ip.h>``          |
++----------------------+-------------------+------------------------------------------------+
+|   Graph              | ``xrt::graph``    |  ``#include <experimental/aie.h>``             |
+|                      |                   |                                                |
+|                      |                   |  ``#include <experimental/graph.h>``           |
++----------------------+-------------------+------------------------------------------------+
 
+Majority of the core data structures are defined inside in the header files at ``$XILINX_XRT/include/xrt/`` directory. Few newer features such as ``xrt::ip``, ``xrt::aie`` related header files are inside ``$XILINX_XRT/include/experimental`` directory. The API interfaces that are in the experimental folder are subject to change in future. 
```

Comment:
All APIs are subject to change and will indeed change, but only the ones in experimental are subject to *breaking* changes.

Reply:
done